test/parallel_exec.jl

```diff
@@ -774,6 +774,17 @@ end
 # Test pmap with a generator type iterator
 @test [1:100...] == pmap(x->x, Base.Generator(x->(sleep(0.0001); x), 1:100))
 
+# Test pgenerate
+a = rand(4,4)
+b = fill(a, 4)
+c = collect(Base.pgenerate(svdfact, b))
```

Comment:
oooooh nice! will do :)

I can change the input arg to have different elements to make sure the return order is correct as well